master/internal/api_experiment.go

```diff
@@ -2105,6 +2105,13 @@ func sortExperiments(sortString *string, experimentQuery *bun.SelectQuery) error
 			experimentQuery.OrderExpr(
 				fmt.Sprintf("trials.summary_metrics->'avg_metrics'->'%s'->>'%s' %s", metricName,
 					metricQualifier, sortDirection))
+		case strings.Contains(paramDetail[0], "."):
```

Comment:
too late at this point, but i think this code would be easier to read nicer if we encoded this as an logical operators (`op: order_by, one: of [{type: trial_summary_metrics, metric_path: []string}, {...}, ...], direction: (asc|desc)`) instead of embedding a small query language.

honestly we are getting to the point here where graphql really makes sense, but im not sure the difficulty of integrating graphql with rbac

Reply:
though, re:graphql, decisions in general.., after making a call, it is easy to see the all the points where a decision (not to use it) becomes suboptimal because they become concrete, where as the hypothetical counterpoints at the time of decision making remain that way.

Reply:
which i guess is obvious, you learn more about the path you take. </rant>

Reply:
I agree it's not ideal.. but I added an utility function that handles parsing, which can be a starting point if we want to refactor.